pkg/provision/buildroot.go

```diff
@@ -155,6 +168,14 @@ WantedBy=multi-user.target
 	}, nil
 }
 
+func rootFileSystemType(p *BuildrootProvisioner) (string, error) {
+	fs, err := p.SSHCommand("df --output=fstype / | sed 1d")
```

Comment:
As I mentioned in the other bug report, why not use `stat --file-system --format '%T' /`?

Reply:
Good idea!